webapp/locales/de.json

```diff
@@ -189,6 +189,25 @@
       "submit": "Link hinzufügen",
       "successAdd": "Social-Media hinzugefügt. Profil aktualisiert!",
       "successDelete": "Social-Media gelöscht. Profil aktualisiert!"
+    },
+    "blocked-users": {
+      "name": "Blockierte Benutzer",
+      "explanation": {
+        "intro": "Wenn ein anderer Benutzer von dir blockiert wurde, dann passiert folgendes:",
+        "your-perspective": "In deiner Beitragsübersicht tauchen keine Beiträge der blockierten Person mehr auf.",
+        "their-perspective": "Umgekehrt das gleiche: Die blockierte Person sieht deine Beiträge auch nicht mehr in ihrer Übersicht.",
+        "search": "Die Beiträge von blockierten Personen verschwinden aus deinen Suchergebnissen.",
+        "notifications": "Von dir blockierte Personen erhalten keine Benachrichtigungen mehr, wenn sie in deinen Beiträgen erwähnt werden.",
+        "closing": "Das sollte fürs Erste genügen, damit blockierte Benutzer dich nicht mehr länger belästigen können."
+      },
+      "columns": {
+        "name": "Name",
+        "slug": "Alias"
```

Comment:
Why here `Alias` and in English `Slug`? @roschaefer 

In the Admins Users list in both languages the slug is name `Slug`.
`Alias`is better understandable for the users for sure!

Reply:
I thought that "Slug" is not so common in German for most people to understand. I don't know what it actually translates to, I believe it's "Schnecke" and refers to the pattern of strings that have no witespaces as delimiter (in a URL).

Good luck understanding that one..

Reply:
But does an English user understands `Slug` better as a German? @roschaefer

Reply:
No, I think "Slug" is also a bad name. Shall we use "alias" everywhere?

Reply:
I understand what a slug is, but can't remember when I gained that understanding :woman_shrugging: 
An alias, I guess is better understood by the general user, but it's not really the same, is it?